src/PrestaShopBundle/Controller/Api/ApiController.php

```diff
@@ -29,13 +29,14 @@
 use Exception;
 use PrestaShopBundle\Api\QueryParamsCollection;
 use Psr\Log\LoggerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
 use Symfony\Component\DependencyInjection\ContainerInterface;
 use Symfony\Component\HttpFoundation\JsonResponse;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
 use Symfony\Component\HttpKernel\Exception\HttpException;
 
-abstract class ApiController
+abstract class ApiController extends Controller
```

Comment:
ApiController should be Symfony controller IMO 🤔  this provides the ability for module and shop developers to benefit from Symfony built-in feature